libs/orchestrator-ui-components-storybook/src/components/Table/DataGridTable.stories.tsx

```diff
@@ -1,14 +1,17 @@
 import type { Meta } from '@storybook/react';
 import React, { useState } from 'react';
-import { DataSorting, SortDirection } from './dataGridColumns';
-import { DataGridTable } from './DataGridTable';
+import {
+    DataSorting,
+    SortDirection,
+    DataGridTable,
+} from '@orchestrator-ui/orchestrator-ui-components';
 
-const extractedArr = (arr, start, howMany) =>
-    arr.filter((_, index) => {
+const extractedArr = (arr: Array<any>, start: number, howMany: number) =>
+    arr.filter((_: any, index: number) => {
         return index >= start && index < howMany + start;
     });
 
-const TableWithEvents = (args) => {
+const TableWithEvents = (args: any) => {
```

Comment:
Isn't the type of args just "DataGridTableProps", that can be imported from the DataGridTable component (this remark counts for all similar story book cases)